File: SantaFinder.Web/Models/OrderHistory/OrderShortInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using SantaFinder.Data.Entities;

namespace SantaFinder.Web.Models.OrderHistory
{
    public class OrderShortInfo
    {
        public int Id { get; set; }
        public DateTime Datetime { get; set; }
        public Address Address { get; set; }
        public string ChildrenNames { get; set; }
        public OrderStatus Status { get; set; }
        public SantaShortInfo SantaInfo { get; set; }

    }
}

File: SantaFinder.Web/Client/src/app/client/order-history/order-short-info.ts
import { Santa } from '../../core/models';
import { OrderStatus } from '../../core/enums';

export interface OrderShortInfo {
    id: number;
    datetime: string;
    address: {
        city: string;
        street: string;
        house: string;
        apartment: string;
    };
    childrenNames: string;
    santaInfo: Santa;
    status: OrderStatus;
}
